nrfu_tests/test_definition.yaml

```
@@ -0,0 +1,13 @@
+- name: nrfu_tests
+  testcases:
+  - name: test_bad_syslog_events
```

```suggestion
  - name: test_system_hardware_bad_syslog_event
```
can we update update function like this ?

same for .py file name